package.json

```diff
@@ -1,7 +1,7 @@
 {
   "main": "node_modules/expo/AppEntry.js",
   "dependencies": {
-    "@react-native-community/datetimepicker": "^3.0.9",
+    "@react-native-community/datetimepicker": "3.0.4",
```

Comment:
So, I know this is bringing us back to 3.0.4. I really don't know if that would be the cause of Naomi's problem with the datetimepicker that she was having, I just assumed that it would be a good place to start with nothing else being am obvious clue.